spidermon/contrib/actions/sentry/__init__.py

```diff
@@ -100,15 +102,47 @@ def get_message(self):
 
         return message
 
+    @staticmethod
+    def get_tags(message):
+        tags = {}
+        failed_monitors = message.get("failed_monitors", [])
+        failed_monitors = [y.split("/") for y in failed_monitors]
+        for key, group in groupby(failed_monitors, key=lambda x: x[0]):
+            for mon in group:
+                try:
+                    k = mon[1].lower().replace(" ", "_")
+                    # 32 chars undocumented tag key length
+                    tags.update({k[:32]: 1})
+                except IndexError:
+                    pass
+        return tags
+
+    @staticmethod
+    def get_reason_hash(message):
```

Comment:
**suggestion**: Albeit this tag can help to group notification with the same failing monitors, maybe this is not general enough and could be better accomplished as a custom action inherited from this action.

Reply:
Yeah, no problem with that. There are also some other ways to configure merging documented at https://docs.sentry.io/product/sentry-basics/guides/grouping-and-fingerprints/